config/locales/en/map.yml

```diff
@@ -63,6 +63,7 @@ en:
         name: "Name"
         message: "Questions / Comments"
         submit: "Confirm"
+        emailInvalidErrorMessage: "Please enter a valid email"
```

Comment:
In these translation files we usually use `snake_case` and also something more succinct for the key. Could you change this to something like `email_invalid`

And can you also add this line to the `map.yml` files for the other locales, and just add "# TODO: Translate" to the end of each line in the other locales.